Bottom: 26085845e1eeb462139b5ae4aec7092817945dc6
Top:    698528ab04b40aabc816db8bd16c1480acf4007b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-24 17:45:26 +0000

Refresh of add-configure-checks-for-gtkgl

---

diff --git a/configure.ac b/configure.ac
index 0908316..3548e63 100644
--- a/configure.ac
+++ b/configure.ac
@@ -281,17 +281,17 @@ Either do not use --enable-gl or enable the gtk HID.])
 # FIXME... what do we need to check for determining we have GL? Is it headers, API ?
 
 	AC_CHECK_HEADERS(GL/gl.h)
-	case $ac_cv_header_GL_glut_h in
+	case $ac_cv_header_GL_gl_h in
 	  no )
 	    AC_MSG_ERROR([You don't seem to have the GL library headers installed.])
 	    ;;
 	  * ) ;;
 	esac
 
-	AC_CHECK_HEADERS(GL/glut.h)
-	case $ac_cv_header_GL_glut_h in
+	AC_CHECK_HEADERS(GL/glu.h)
+	case $ac_cv_header_GL_glu_h in
 	  no )
-	    AC_MSG_ERROR([You don't seem to have the GL glut library headers installed.])
+	    AC_MSG_ERROR([You don't seem to have the GL glu library headers installed.])
 	    ;;
 	  * ) ;;
 	esac
